agent/agentssh/agentssh.go

```diff
@@ -322,6 +341,35 @@ func (s *Server) sessionHandler(session ssh.Session) {
 	_ = session.Exit(0)
 }
 
+// fileTransferBlocked method checks if the file transfer commands should be blocked.
+// Warning: consider this mechanism as "Do not trespass" sign. If a user needs a more sophisticated
+// and battle-proof solution, consider the full endpoint security.
```

Comment:
Minor grammatical nit.

```suggestion
// and battle-proof solution, consider  full endpoint security.
```